### PR TITLE
Line edits

### PR DESCRIPTION
**Pull request by rachfop**
## What does this PR do?
First pass of line edits.

### CODE REVIEW

File: docs/typescript/introduction.md [hunk 1]
```diff
@@ -59,12 +59,12 @@ Run "Hello Temporal" locally (~10 minutes)
 Run Temporal Server (requires [Docker](https://docs.docker.com/engine/install) and [Docker Compose](https://docs.docker.com/compose/install/)):
 
 ```bash
-git clone https://github.com/temporalio/docker-compose.git temporal
```
Comment (lorensr) [inline docs/typescript/introduction.md:62]: `temporal` is intentional (naming the local directory)

File: docs/typescript/index.md [hunk 2]
```diff
@@ -11,24 +11,6 @@ tags:
...
-npm i @temporalio/client @temporalio/worker @temporalio/workflow @temporalio/activity
-```
-
-Or clone the TypeScript SDK repo to your preferred location:
-
-```bash
-git clone git@github.com:temporalio/sdk-typescript.git
-```
-
-This project requires Node.js 14 or later.
```
Comment (lorensr) [inline docs/typescript/index.md:26]: Is this line elsewhere? (want to retain somewhere)

File: docs/application-development/foundations.md [hunk 3]
```diff
@@ -282,9 +271,7 @@ The Temporal TypeScript SDK API reference is published on [typescript.temporal.i
 
 #### Code samples
 
-You can find a complete list of executable code samples in [Temporal's GitHub repository](https://github.com/temporalio).
-
-The [Temporal Simple Polyglot](https://github.com/temporalio/temporal-polyglot) repository showcases how Workflow Executions, written in different languages, can send messages to each other. Go, Java, PHP, and TypeScript SDKs are represented in this sample. It also shows how to properly propagate errors, including how to do so across Workflows written in different languages. For more information, see the [Polyglot Microservice Orchestration](https://www.youtube.com/playlist?list=PLl9kRkvFJrlTLo5URV5IK6lCmiM3ir3f5) video on YouTube.
+You can find a complete list of executable code samples in [Temporal's GitHub repository](https://github.com/temporalio?q=samples-&type=all&language=&sort=).
```
Comment (lorensr) [inline docs/application-development/foundations.md:274]: don't think this link is helpful, would remove and change below line to:

In addition to the samples listed in the language tabs below, several of the [Tutorials](https://learn.temporal.io) are backed by working applications.

or remove that too and list tutorials in the lang tabs

**Review by lorensr — APPROVED** [review]

**Review by flossypurse — APPROVED** [review]

**Review by djmagee — APPROVED** [review]
Just two grammar nits.

File: docs/app-dev-context/code-samples.md [hunk 4]
```diff
@@ -7,8 +7,6 @@ tags:
   - guide-context
 ---
 
-You can find a complete list of executable code samples in [Temporal's GitHub repository](https://github.com/temporalio).
-
-The [Temporal Simple Polyglot](https://github.com/temporalio/temporal-polyglot) repository showcases how Workflow Executions, written in different languages, can send messages to each other. Go, Java, PHP, and TypeScript SDKs are represented in this sample. It also shows how to properly propagate errors, including how to do so across Workflows written in different languages. For more information, see the [Polyglot Microservice Orchestration](https://www.youtube.com/playlist?list=PLl9kRkvFJrlTLo5URV5IK6lCmiM3ir3f5) video on YouTube.
```
[the remark this answers is not in the document]
Comment (bergundy) [reply docs/app-dev-context/code-samples.md:12]: +1

File: docs/app-dev-context/replays.md [hunk 5]
```diff
@@ -11,14 +11,3 @@ Replay recreates the exact state of a Workflow Execution.
 You can replay a Workflow from the beginning of its history when resumed.
 
 Replay allows code to resume only if it is compatible from a deterministic point of view.
-
-To retrieve the Workflow History, use any of the following options and then pass the object to your SDK of choice.
```
Comment (bergundy) [inline docs/app-dev-context/replays.md:15]: This actually good content

File: docs/application-development/foundations.md [hunk 6]
```diff
@@ -241,6 +224,12 @@ npx @temporalio/create@latest ./your-app
 npm install @temporalio/client @temporalio/worker @temporalio/workflow @temporalio/activity @temporalio/common
 ```
 
+:::note
+
+The TypeScript SDK is designed with TypeScript-first developer experience in mind, but works equally well with JavaScript.
```
[the remark this answers is not in the document]
Comment (bergundy) [reply docs/application-development/foundations.md:229]: I’d add that node 14 is required somewhere

**Review by bergundy — APPROVED** [review]
Had a few minor suggestions, thanks!

**Pull request merged**